### DISCUSSION TITLE
Better release announcements / documentation

### DISCUSSION THREAD

**Discussion by renchap** [yarnpkg/berry#3186] [Ideas]
I learned that Yarn 3 was released through a comment on an issue in another repo.

As far as I can see, there are no Github releases for new versions, and no blog (or RSS feed) on Yarn's website, no Twitter, no mailing-list.

It would really help users to have a proper release announcement channel. Github releases are probably the easiest way to do it, as you can easily watch the project's releases and get notified (+ they are now displayed in your homepage's feed).

For major new versions it would also be awesome to have an official announcement outlining the major changes, in addition to the current changelog.

**Comment by danoc** [comment]
The announcements are published here:
https://dev.to/arcanis/yarn-3-0-performances-esbuild-better-patches-e07

I agree that it'd be great if it were more discoverable though.

**Reply by Kurt-von-Laven** [reply]
They are also 3 minor versions out of date at this point as of today's release of v3.5.0. It would be great to have posts for v3.3, v3.4, and v3.5. It might help to host them in the GitHub release notes since, as it stands, it's all on one person to write all of the release notes by themselves.

**Comment by merceyz** [comment]
As an update to this: We've been creating [GitHub Releases](https://github.com/yarnpkg/berry/releases) for a while now.

**Reply by Kurt-von-Laven** [reply]
That was my point; the ask is for release notes, not releases, though.
